packages/synapse-react-client/src/utils/functions/queryUtils.test.ts

```diff
@@ -28,15 +23,19 @@ import {
   getCorrespondingSelectedFacet,
   getHeaderIndex,
   hasResettableFilters,
-  isFacetAvailable,
+  hasFacetedSelectColumn,
   isSingleNotSetValue,
   queryRequestsHaveSameTotalResults,
   removeEmptyQueryParams,
-} from '../../src/utils/functions/queryUtils'
-import { mockTableEntity } from '../../src/mocks/entity/mockTableEntity'
-import mockDataset from '../../src/mocks/entity/mockDataset'
-import { mockFileViewEntity } from '../../src/mocks/entity/mockFileView'
-import mockDatasetCollection from '../../src/mocks/entity/mockDatasetCollection'
+  removeLockedColumnFromFacetData,
+} from './queryUtils'
+import { mockTableEntity } from '../../mocks/entity/mockTableEntity'
+import mockDataset from '../../mocks/entity/mockDataset'
+import { mockFileViewEntity } from '../../mocks/entity/mockFileView'
+import mockDatasetCollection from '../../mocks/entity/mockDatasetCollection'
+import * as SynapseConstants from '../SynapseConstants'
+import { LockedColumn, UniqueFacetIdentifier } from '../types'
+import mockQueryResponseData from '../../mocks/mockQueryResponseData'
 
 describe('facet support', () => {
```

Comment:
Moved this file so it is now type-checked. Moved some of the LockedColumn tests here since the utility method was extracted here